index.html

```diff
@@ -63,6 +63,150 @@ <h3 class="secondaryText">
+            </div>
+        </div>
+
+        <div class="card workCard">
+            <div class="imagePlaceHolder"></div>
+            <div class="workInfo">
+                <h3  class="cardHeader">Multi-Post Stories <br>
+                    Gain + Glory</h3>
+                <ul class="tagsHandler workTags">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li>HTML</li>
+                </ul>
+                <button type="button" class="button">See Project</button>
+            </div>
+        </div>
+
+        <div class="card workCard">
+            <div class="imagePlaceHolder"></div>
+            <div class="workInfo">
+                <h3  class="cardHeader">Multi-Post Stories <br>
+                    Gain + Glory</h3>
+                <ul class="tagsHandler workTags">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li>HTML</li>
+                </ul>
+                <button type="button" class="button">See Project</button>
+            </div>
+        </div>
+
+        <div class="card workCard">
+            <div class="imagePlaceHolder"></div>
+            <div class="workInfo">
+                <h3  class="cardHeader">Multi-Post Stories <br>
+                    Gain + Glory</h3>
+                <ul class="tagsHandler workTags">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li>HTML</li>
+                </ul>
+                <button type="button" class="button">See Project</button>
+            </div>
+        </div>
+
+        <div class="card workCard">
+            <div class="imagePlaceHolder"></div>
+            <div class="workInfo">
+                <h3  class="cardHeader">Multi-Post Stories <br>
+                    Gain + Glory</h3>
+                <ul class="tagsHandler workTags">
+                    <li>Ruby on Rails</li>
+                    <li>CSS</li>
+                    <li>JavaScript</li>
+                    <li>HTML</li>
+                </ul>
+                <button type="button" class="button">See Project</button>
+            </div>
+        </div>
+
+    </section>
+
+    <section class="aboutme centeredGrid">
+        <div class="sectionHeader aboutmeHeader">
+            <h2>About me</h2>
+            <h3> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</h3>
+            <button type="button" class="button">Get My Resume</button>
+        </div>
+        
+
+        <div class="card aboutmeCard">
+            <img src="./src/aboutme/icon-languages.svg" alt="">
+            <h3 class="cardHeader aboutmeHeader">Languages</h3>
```

Comment:
- [ ] Please stick to the [Figma ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) Design. Your `h3` tag does not need any background, it will not make it pretty. Make it simple and beautiful 😃. 
- The design you did.
 ![ ](https://res.cloudinary.com/dflkwvvnz/image/upload/v1628828121/scrnli_8_13_2021_9-13-49_AM_mjqq7g.png)